snuba/datasets/configuration/generic_metrics/storages/org_counters.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+kind: readable_storage
+name: generic_org_metrics_counters
+storage:
+  key: generic_org_metrics_counters
+  set_key: generic_metrics_counters
+schema:
+  columns:
+    [
+      { name: org_id, type: UInt, args: { size: 64 } },
+      { name: project_id, type: UInt, args: { size: 64 } },
+      { name: metric_id, type: UInt, args: { size: 64 } },
+      { name: granularity, type: UInt, args: { size: 8 } },
+      { name: timestamp, type: DateTime },
```

Comment:
is this missing the actual count column?

https://github.com/getsentry/snuba/blob/master/snuba/datasets/configuration/generic_metrics/storages/counters.yaml#L48-L56

or do you just not need it?

Reply:
I think we need it here, @evanh  @untitaker wdyt? 
I'm not very familiar with how this will work under the hood.

Reply:
only i can say it works with and without this block. i just did quick test.

Reply:
I see, yeah I prefer to implement as minimally as we need. So later we can extend.